sinch/domains/numbers/api/v1/internal/active_numbers_endpoints.py

```diff
@@ -1,5 +1,5 @@
 from sinch.core.models.http_response import HTTPResponse
-from sinch.domains.numbers.api.v1.internal.numbers_endpoint import NumbersEndpoint
+from sinch.domains.numbers.api.v1.internal.base.numbers_endpoint import NumbersEndpoint
```

Comment:
This import could be simplified with
```suggestion
from sinch.domains.numbers.api.v1.internal.base import NumbersEndpoint
```

Reply:
done